README.MD

```diff
@@ -14,15 +16,15 @@ Perhaps, Python🐍 can help you out to apply those mesmerizing filters with som
 
 <h2 align = "center">Demo</h2>
 
-|            Work Done         |           Original Image**    |           Output Image                |
+|            Work Done         |           Original Image      |           Output Image                |
```

Comment:
This is done for giving credits to the photographer/source of the image used

Reply:
Okay! 
So I shouldn't change this. 
And everything else is okay??

Reply:
Yes the remaining changes are fine

Reply:
Okay!
So, since it's like I'm doing these contributions and all for the first time. Should I create another branch and then edit and push to GitHub or what? 
Can you guide me a bit with this?

Reply:
Ideally, you should create a feature branch for the change, push it to your remote on GitHub and give a PR. But for documentation changes, there is usually no merge conflicts arising due to other's commits, so you can do the changes in your master branch as well

Reply:
Oh okay. 
So I should just go to my forked repo of yours and then in master only change this thing and give it a try, right?

Reply:
Ya. that works for this PR

Reply:
You can look at it now!